src/crypto/tls/generate_cert.go

```diff
@@ -32,6 +32,7 @@ var (
 	validFrom  = flag.String("start-date", "", "Creation date formatted as Jan 1 15:04:05 2011")
 	validFor   = flag.Duration("duration", 365*24*time.Hour, "Duration that certificate is valid for")
 	isCA       = flag.Bool("ca", false, "whether this cert should be its own Certificate Authority")
+	isDC       = flag.Bool("dc", false, "whether this cert can be used with delegated credentials")
```

Comment:
Perhaps: allowDC?  Or forDC?

Reply:
I like `allowDC`. I will look like this: `go run generate_cert.go -ecdsa-curve P256 -host 127.0.0.1 -allowDC`